lib/intercom-rails/script_tag.rb

```diff
@@ -26,6 +27,19 @@ def initialize(options = {})
       self.nonce = options[:nonce]
     end
 
+    def remove_user_cookie_on_logout
+      if (find_current_user_details == {} && controller.response.request.cookies["intercom-session-#{IntercomRails.config.app_id}"])
+        controller.response.set_cookie("intercom-session-#{IntercomRails.config.app_id}", :value => "", :expires => Time.at(0))
```

Comment:
controller.response is an instance of [ActionDispatch::Response](http://api.rubyonrails.org/classes/ActionDispatch/Response.html) which doesn't implement delete_cookie. The best way to erase a cookie without a delete_cookie method is to set a value of nil with an expiration date in the past.